pkg/pillar/scripts/device-steps.sh

```diff
@@ -337,6 +345,9 @@ access_usb() {
 # Read any usb.json with DevicePortConfig, and deposit our identity
 access_usb
 
+# Update our local /etc/hosts with entries comming from /config
+[ -f /config/hosts ] && cat /config/hosts >> /etc/hosts
```

Comment:
This will append to /etc/hosts for every boot so the file will grow and grow.

Reply:
That's actually not the case. /etc/hosts inside of pillar container is synthesized from scratch during every boot. It is not persistent and has to be appended to on every boot.

Makes sense @nordmark ?

Reply:
Ping @nordmark  ?